sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -183,11 +183,14 @@
   color: $brand-blue;
 }
 
-.formio-component .fa {
+.formio-component .fa::before {
```

Comment:
for my info, why pseudo-element?

Reply:
It is to apply the font awesome as font family to the first child only so that wordings follow the default font-family.